Modules/socketmodule.c

```diff
@@ -7737,6 +7737,17 @@ PyInit__socket(void)
 #else
     PyModule_AddIntConstant(m, "IPPROTO_UDP", 17);
 #endif
+#ifdef  IPPROTO_UDPLITE
+    PyModule_AddIntMacro(m, IPPROTO_UDPLITE);
+    #ifndef UDPLITE_SEND_CSCOV
+        #define UDPLITE_SEND_CSCOV 10
```

Comment:
I believe if `IPPROTO_UDPLITE` is defined then `UDPLITE_SEND_CSCOV` / `UDPLITE_RECT_CSCOV` are defined too.
What is the reason to use custom numeric constants?

Reply:
When I tried to compile on my system it hit an error unless I added that condition. I'm not sure how common that is, but it happened to me on Ubuntu 18.04.02. If it happened to me, I assume there are other systems where it will happen. From reading the manpage it seems that there's varying glibc support for those macros, and they had a section on how they should be defined if they weren't present.